packages/backend/src/nest/storage/storage.service.ts

```diff
@@ -314,92 +309,41 @@ export class StorageService extends EventEmitter {
     }
 
     try {
-      await this.certificates?.close()
+      await this.certificatesStore?.close()
     } catch (e) {
       this.logger.error('Error closing certificates db', e)
     }
+
     try {
       await this.certificatesRequests?.close()
     } catch (e) {
       this.logger.error('Error closing certificates db', e)
     }
+
     try {
       await this.communityMetadata?.close()
     } catch (e) {
       this.logger.error('Error closing community metadata db', e)
     }
+
     await this.__stopOrbitDb()
     await this.__stopIPFS()
   }
 
-  public async updatePeersList() {
-    const allUsers = this.getAllUsers()
-    const registeredUsers = this.getAllRegisteredUsers()
-    const peers = [...new Set(await getUsersAddresses(allUsers.concat(registeredUsers)))]
-    console.log('updatePeersList, peers count:', peers.length)
-    const community = await this.localDbService.get(LocalDBKeys.COMMUNITY)
-    this.emit(StorageEvents.UPDATE_PEERS_LIST, { communityId: community.id, peerList: peers })
-  }
+  public async updatePeersList() {}
```

Comment:
We no longer need a list of certificates to update peers list (we use csrs instead) so to make things simpler for merging with https://github.com/TryQuiet/quiet/pull/2071 I removed certain methods so we can just accept incoming changes when resolving conflicts